package.json

```diff
@@ -49,6 +50,7 @@
     "electron-store": "8.2.0",
     "electron-window-state": "5.0.3",
     "lodash": "^4.17.21",
+    "n": "^10.2.0",
```

Comment:
Can you remove this module? is not related to the app. Their docs do recommend installing it globally, not locally. Thanks!

(I use nvm but I am not curious how it compares :) )